### PR TITLE
Adding bio file for kmunsell

### PR DESCRIPTION
**Pull request by kmunsell**
This pull request is supposed to fix the issue I @kmunsell opened #7

### CODE REVIEW

**Comment by kmunsell** [comment]
@jrevans checking to see if I did this correctly

File: kmunsell-bio-md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Kirk Munsell Bio
+
+## About Me Today
+
+- Taking intro class on GitHub
+- Working on my bio description
+
```
Comment (jrevans) [inline kmunsell-bio-md]: Looks good, but what is your favorite color?

**Comment by kmunsell** [comment]
@jrevans I'll add my color now

**Comment by kmunsell** [comment]
@githubteacher  here's a new mention

**Pull request merged**